common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -317,6 +324,32 @@ protected void processRedirectUrl(@NonNull final WebView view, @NonNull final St
         //the TokenTask should be processed at after the authorization process in the upper calling layer.
     }
 
+    /**
+     * Launch the browser with the given action URI and code.
+     * <p>
+     * From the query parameters, extract the action URI and code,
+     * The constructs the URI with the action URI and code.
+     *
+     * @param uri The URI of the request.
+     */
+    protected boolean processSwitchToBrowserRequest(@NonNull final Uri uri) {
```

Comment:
Suggest removing the `to` from the method name to remain consistent with the naming used for the feature and easier discoverability.

```suggestion
    protected boolean processSwitchBrowserRequest(@NonNull final Uri uri) {
```